app/src/main/java/com/asfoundation/wallet/ui/iab/MergedAppcoinsFragment.kt

```diff
@@ -377,11 +382,16 @@ class MergedAppcoinsFragment : DaggerFragment(), MergedAppcoinsView {
   }
 
   override fun showBonus() {
-    val animation = AnimationUtils.loadAnimation(context, R.anim.fade_in_animation)
-    animation.duration = 250
-    bonus_layout?.visibility = VISIBLE
-    bonus_layout?.startAnimation(animation)
-    bonus_msg?.visibility = VISIBLE
+    if (bonus.isNotBlank()) {
```

Comment:
The same comment as above, regarding the possibility of using isNotEmpty instead of isNotBlank.